server/templates/home.html

```diff
@@ -35,106 +58,227 @@
                     </div>
                 </div>
             </form>
-            <div class="logout col-sm-2">
-                    <button class="btn btn-default pull-right logout-button">
-                            <span class="glyphicon glyphicon-log-out"></span> Logout
-                    </button>
-            </div>
         </div>
 
-        <!-- Table showing selected sample files -->
-        <div class="row col-sm-10 col-sm-offset-1 sample-table">
-            <div class="panel panel-default">
-                <div class="panel-heading">
-                    <span class="collapse-icon glyphicon glyphicon-triangle-bottom pull-left"></span>
-                    <span class="collapse-icon glyphicon glyphicon-triangle-right pull-left" style="display: none;"></span>
-                    Select Samples for Upload
-                </div>
-                <div class="panel-body">
-                    <p>
-                        To start uploading your files hit the <em>Add Sample</em> button. You will be required to
-                        provide a sample name and specify all applicable metadata for the sample you wish to upload.
-                        You may also alter sample and individual metadata options after attaching your files. Sample
-                        name, readset, and library must be composed solely of letters, numbers, hyphens, and/or
-                        underscores to ensure compatibility with downstream pipelines.
-                    </p>
-                    <div data-toggle="tooltip" title="Add a new sample">
-                        <button class="btn btn-primary pull-right modal-add-sample-button"
-                                data-toggle="modal"
-                                data-target="#add-sample-modal">
-                            <span class="glyphicon glyphicon-plus"></span>
-                            Add Sample
-                        </button>
+        <div class="row">
+            <div class="main col-sm-10 col-sm-offset-1 col-md-10 col-md-offset-1">
+
+              <!-- Nav tabs -->
+                <ul class="nav nav-tabs" role="tablist">
+                    <li role="presentation" class="active">
+                        <a class="tab-label" href="#upload" role="tab" data-toggle="tab">Upload</a>
+                    </li>
+                    <li role="presentation">
+                        <a class="tab-label" href="#manage" role="tab" data-toggle="tab">Manage</a>
+                    </li>
+                    <li role="presentation">
+                        <a class="tab-label" href="#analyze" role="tab" data-toggle="tab">Analyze</a>
+                    </li>
+                </ul>
```

Comment:
Let's take this opportunity to split up these modals, tables, etc. into separate html fragment files that get included. Take a look at the `include` template and `import` macro sections of the Jinja documentation for examples: http://jinja.pocoo.org/docs/dev/templates/#include